src/xdocs/config_filters.xml

```diff
@@ -2232,6 +2232,275 @@ public class UserService {
+          To configure the filter to suppress audit events on any line that contains
+          <code>DO NOT CHECK THIS LINE</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;DO NOT CHECK THIS LINE&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 42; // DO NOT CHECK THIS LINE
+int b = 43; // violation
+        </source>
+        <p>
+           To configure the filter to suppress audit events whose check message contains
+            the word <code>Line</code>. In this case, <code>LineLengthCheck</code>'s violation
+            message contains it:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;messagePattern&quot; value=&quot;.*Line.*&quot;/&gt;
+    &lt;property name=&quot;nearbyTextPattern&quot; value=&quot;.*&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;LineLength&quot;&gt;
+    &lt;property name=&quot;max&quot; value=&quot;10&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+export FOO=BAR # ok, because violation message is matching suppress pattern
+        </source>
+        <p>
+          To configure the filter to suppress audit events only on a check whose id is
+          <code>ignoreMe</code>:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;idPattern&quot; value=&quot;ignoreMe&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;LineLength&quot;&gt;
+    &lt;property name=&quot;max&quot; value=&quot;15&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MagicNumber&quot;&gt;
+      &lt;property name=&quot;id&quot; value=&quot;ignoreMe&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+int a = 42; // ok
+int longVarName = 2; // LineLength violation
+        </source>
+        <p>
+          To configure the filter to suppress audit events for the current and next 2 lines:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;SuppressWithNearbyTextFilter&quot;&gt;
+    &lt;property name=&quot;lineRange&quot; value=&quot;2&quot;/&gt;
+  &lt;/module&gt;
+  &lt;module name=&quot;UniqueProperties&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example:
+        </p>
+        <source>
+key.one=41 # SUPPRESS CHECKSTYLE the reason for this is
```

Comment:
`the reason for this is` --> `because I want`

fix for all others.

Reply:
Done.